CONTRIBUTING.md

```diff
@@ -112,13 +112,17 @@ Just make sure to remove the dependencies again before finally making a pull req
 ### Rust Analyzer
 As of [#6869][6869], [`rust-analyzer`][ra_homepage] can understand that Clippy uses compiler-internals
 using `extern crate` when `package.metadata.rust-analyzer.rustc_private` is set to `true` in Clippy's `Cargo.toml.`
+
 You will require a `nightly` toolchain with the `rustc-dev` component installed.
+Notice that [rust-toolchain](./rust-toolchain) pins a specific nightly version, but for the
+configuration below to work you will need `rustc-dev` for the **regular** nightly.
```

Comment:
I might be confusing the wording here. Is this trying to say that: it's ok if `rustc-dev` version used is not the same as the one in `rust-toolchain`?

I guess this being placed after L116 is what making me confused.

Reply:
This is trying to say that the `rustc-dev` version that is automatically installed through the `rust-toolchain` file isn't enough and that you need a `rustc-dev` component in a `nightly` toolchain.

However, I don't think this is the case, because it works for me and I don't have a `rustc-dev` component in my `nightly` toolchain.

Reply:
Thanks! I undid the change. I explained a little bit more [in the other thread](https://github.com/rust-lang/rust-clippy/pull/9172#discussion_r921580969), but it was likely due to some mistake I made during the initial setup.